prdoc/pr_4889.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add CLI options for parachain chain specifications + fix bug for swallowing custom fields
+
+doc:
+  - audience: Node Operator
```

Comment:
oooh really? I think it is Node Dev

Reply:
I was conflicted about this, I copied some previous PRs on chain-spec-builder, you guys would know better than me about it 🙈

Reply:
If it was me specifying *operator*, it is because node operator may sometime play with this util, during chain setup. But I may be wrong.
Node-dev also may never need to generate chain-spec.